orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -82,6 +82,24 @@ public TypeDescription type() {
 
   static final String ICEBERG_FIELD_LENGTH = "iceberg.length";
 
+  /**
+   * The name of the ORC {@link TypeDescription} attribute indicating the Iceberg timestamp unit.
```

Comment:
nit: the two descriptions are mixed up (`iceberg.timestamp-unit`, `iceberg.struct-type`)

Reply:
Fixed. Thanks for catching this!